### PR TITLE
Convert QrCodeView component to use JSX

### PR DESCRIPTION
**Pull request by whymarrh**
Refs #6291

This PR replaces hyperscript in the `QrCodeView ` component with JSX.

### CODE REVIEW

File: ui/app/components/ui/qr-code.js [hunk 1]
```diff
@@ -17,47 +16,60 @@ function mapStateToProps (state) {
...
+      <div
+        className="div qr-wrapper"
+        dangerouslySetInnerHTML={{
+          __html: qrImage.createTableTag(4),
+        }}
+      />
+      <ReadOnlyInput
+        wrapperClass="ellip-address-wrapper"
+        inputClass="qr-ellip-address"
+        value={checksumAddress(data, network)}
```
Comment (Gudahtt) [inline ui/app/components/ui/qr-code.js]: I'm guessing that `network` isn't needed here either?

Comment (whymarrh) [reply ui/app/components/ui/qr-code.js]: Yeah, good catch

Comment (whymarrh) [reply ui/app/components/ui/qr-code.js]: I had removed it from the `propTypes` when I saw that it wasn't functionally used

Comment (whymarrh) [reply ui/app/components/ui/qr-code.js]: Updated

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**